controllers/provider-openstack/example/infrastructure.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: shoot--foo--bar
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: garden.sapcloud.io:system:terraformers
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - '*'
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: core-openstack
+  namespace: shoot--foo--bar
+type: Opaque
+data:
+# domainName: base64(domain-name)
+# tenantName: base64(tenant-name)
+# username: base64(username)
+# password: base64(password)
+
+---
+apiVersion: extensions.gardener.cloud/v1alpha1
+kind: Cluster
+metadata:
+  name: shoot--foo--bar
+spec:
+  cloudProfile:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: CloudProfile
+    spec:
+      openstack:
+        keystoneURL: https://localhost
+        # dnsServers:
+        # - 1.1.1.1
+  seed:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: Seed
+  shoot:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: Shoot
+    spec:
```

Comment:
as far as I can see openstack does not need this information (pod/service network). so let's remove it here from the example (whole `spec` section) to reduce surface for confusing why it is here int he first place